docs/source/getting_started/code_examples.rst

```diff
@@ -121,7 +122,7 @@ a parameter template ``wavelength_template``, and a property template ``refracti
     reaction_run = solvents_dataset.process_runs.register(
         ProcessRun("A chemical reaction", spec=reaction_spec))
     toluene_ingredient = solvents_dataset.ingredient_runs.register(
-        IngredientRun("Toluene solvent", spec=toluene_ingredient_spec,
+        IngredientRun(spec=toluene_ingredient_spec,
         material=toluene, process=reaction_run, absolute_quantity=NominalReal(40, 'mL'), notes="I poured too much!"))
 
 Getting a material history
```

Comment:
@kroenlein do you know what a `buy_toluene` is supposed to refer to on line https://github.com/CitrineInformatics/citrine-python/pull/687/files#diff-daad57bd0515ee9846f70e7f9321bea433d4f17671e62b334ffbf76292cac928R148 below?

Reply:
It sounds like `buy_toluene` is supposed to the the process spec that generates the raw material `Toluene`.  Is that the question?